src/cart/crt_swim.c

```diff
@@ -247,7 +247,7 @@ static void crt_swim_srv_cb(crt_rpc_t *rpc_req)
 	rpc_swim_output->rc = rc;
 	rc = crt_reply_send(rpc_req);
 	if (rc)
-		D_ERROR("send reply %d failed "DF_RC"\n",
+		D_ERROR("send reply %d failed " DF_RC "\n",
```

Comment:
There has been some discussion on the use of spaces around DF_RC, I've been writing code with spaces as here but got complaints, so I've just changed to reverting code back to not using spaces, so the opposite of what this PR does here.

Reply:
Checkpatch was giving out warnings without the spaces.  So I put in the spaces to remove the warnings and make a cleaner compile.

Reply:
We need to change checkpatch.

Reply:
I just recheck.  It was another style checking program (don't remember which one) that was causing the warnings.  It wasn't checkpatch.

Reply:
It's checkpatch but it's on of the non-default checks, but we run it in strict mode.

The fix is to add CONCATENATED_STRING to the list at https://github.com/daos-stack/daos/blob/master/ci/patch_src_in_place#L17

This has been on my TODO list for a week or so, I can do it later today.  For this PR though could you back out the changes please?